src/include/sof/math/exp_fcn.h

```diff
@@ -0,0 +1,25 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
+ *
+ * Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+ *
+ */
+#ifndef __SOF_MATH_BASE_EXPONENTIAL_H__
+#define __SOF_MATH_BASE_EXPONENTIAL_H__
+
+/* define constants */
+#define BIT_MASK_Q62P2 4611686018427387904LL
+#define BIT_MASK_LOW_Q27P5 134217728ULL
+#define CONVERG_ERROR 28823037607936LL
```

Comment:
Add a comment or rationale why the magic values in CONVERG_ERROR and QUOTIENT_SCALE. Use hex if it's a bitfield number.